src/utils.ts

```diff
@@ -32,19 +32,20 @@ export const mac = ios || /Mac/.test(platform);
  */
 export type afterDOMUpdateHandle = {
   raf?: number,
-  timeout?: ReturnType<typeof setTimeout>
+  timeout?: {h?:ReturnType<typeof setTimeout>}
 };
 export function setAfterDOMUpdate(f: ()=>void, extraDelay?:number): afterDOMUpdateHandle {
-  let timeoutHandle: ReturnType<typeof setTimeout>;
+  let timeoutHandle = {h: null as ReturnType<typeof setTimeout>};
   return {
-    raf: window.requestAnimationFrame(() => timeoutHandle = setTimeout(f, extraDelay)),
+    raf: window.requestAnimationFrame(() => timeoutHandle.h = setTimeout(f, extraDelay)),
     timeout: timeoutHandle
   };
```

Comment:
I think this is a bit more straightforward if you use the handle as the box.

```ts
const handle: afterDOMUpdateHandle = {
  raf: window.requestAnimationFrame(() => handle.timeout = setTimeout(f, extraDelay))
};
return handle;
```

Reply:
Whoa. That's just...lovely. I didn't even know you could mutate an object when you initialize it!!